webapp/src/websocket.ts

```diff
@@ -156,48 +156,46 @@ export class WebSocketClient extends EventEmitter {
                 this.ws.send(JSON.stringify(msg));
             }
         } else {
-            logWarn('failed to send message, connection is not open');
+            logWarn('failed to send message, connection is not open', msg);
         }
     }
 
-    close(code?: number) {
-        if (this.ws) {
-            this.closed = true;
-            this.ws.close();
-            this.ws = null;
-            this.seqNo = 1;
-            this.serverSeqNo = 0;
-            this.connID = '';
-            this.originalConnID = '';
-
-            this.removeAllListeners('open');
-            this.removeAllListeners('event');
-            this.removeAllListeners('join');
-            this.removeAllListeners('close');
-            this.removeAllListeners('error');
-            this.removeAllListeners('message');
-        } else {
-            this.emit('close', code);
+    close() {
```

Comment:
I suppose we would we have the same problem on mobile?

Reply:
Sent https://github.com/mattermost/mattermost-mobile/pull/7513, great catch :)